Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -363,31 +362,16 @@ export class BotProject {
     }
   }
 
-  private async deleteFilesFromBottomToUp(path) {
-    const files = await this.fileStorage.readDir(path);
-
-    for (let i = 0; i < files.length; i++) {
-      const curPath = Path.join(path, files[i]);
-      const childStat = await this.fileStorage.stat(curPath);
-      if (childStat.isDir && curPath.startsWith(this.dir)) {
-        await this.deleteFilesFromBottomToUp(curPath);
-        await this.fileStorage.rmDir(curPath);
-      } else {
-        await this.fileStorage.removeFile(curPath);
-      }
-    }
-  }
-
```

Comment:
👍 to deleting this.

Reply:
+1, also can you also just delete AzureBLobStorage class which is not used and maintained anymore

Reply:
deleted